omero/training-server/playbook.yml

```diff
@@ -355,6 +393,18 @@
           -A INPUT -p tcp -m tcp --dport 10389 -s 0.0.0.0/0 -j ACCEPT
         state: present
 
+    # TODO: Move to an independent role, currently bundled in
+    # https://github.com/manics/ansible-role-minio-s3-gateway/blob/0.1.0/tasks/minio-client.yml
+    - name: Download minio client
+      become: true
+      get_url:
+        url:
+          https://dl.min.io/client/mc/release/linux-amd64/archive/mc.RELEASE.2020-11-25T23-04-07Z
```

Comment:
Not the fault of this PR, but it'd be nice if there were a slightly more robust way of installing the client.